cadquery/cq.py

```diff
@@ -1468,6 +1468,48 @@ def parametricCurve(self, func, N=400, start=0, stop=1):
 
         return self.spline(allPoints,includeCurrent=False,makeWire=True)
 
+    def ellipseArc(self, x_radius, y_radius, angle1=360, angle2=360, rotation_angle=0.0, sense=1,
+                forConstruction=False, startAtCurrent=True, makeWire=False):
```

Comment:
Open question: do we have a usecase for `startAtCurrent=False`?

Reply:
Good question. I found a similar construct with `spline` (`includeCurrent`) where I would also not see an immediate use case. So having seen this pattern in cq, I decided to add the same approach to `ellipseArc` and let others find out use cases.
But I am also happy to remove if someone has arguments against it.